codeforamerica/vita-min
# FYST-552 add xml & pdf output for mailing and permanent address in maryland

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-552
## Is PM acceptance required?
- Yes - don't merge until JIRA issue is accepted!
**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Added XML & PDF output that 
  - output the DF address to mailing address fields & 
  - output either the DF or the user-entered permanent address to the physical address fields
## How to test?
 - Go to the heroku deployment, 
 - go through the flow confirming your DF address and confirm that it's in the PDF & XML output
 - go through the flow, entering a different address and confirm that it's in the PDF & XML output
- Risk Assessment
  - We did need to edit the MD502 PDF due to a bug with field names that have unmatched parentheses. This means that we may have to edit the TY2024 PDF when we get it as well.
